{"name": "Принятие заказа с различными невалидными ID курьеров", "status": "passed", "steps": [{"name": "Попытаться принять заказ с невалидным ID курьера: None", "status": "passed", "steps": [{"name": "Принять заказ 546576 курьером None", "status": "passed", "parameters": [{"name": "order_id", "value": "546576"}, {"name": "courier_id", "value": "None"}], "start": 1758527548892, "stop": 1758527549261}], "start": 1758527548892, "stop": 1758527549261}, {"name": "Проверить что запрос завершился ошибкой", "status": "passed", "start": 1758527549261, "stop": 1758527549261}], "parameters": [{"name": "invalid_courier_id", "value": "None"}], "start": 1758527548892, "stop": 1758527549263, "uuid": "66d299d3-ae5d-4160-969b-e5dd90296cf9", "historyId": "e536d13040c63794ca5ad9354f3c231e", "testCaseId": "4409f057984de48866aed05d3c5372e9", "fullName": "tests.test_accept_order.TestAcceptOrder#test_accept_order_various_invalid_courier_ids", "labels": [{"name": "epic", "value": "Order Management"}, {"name": "severity", "value": "normal"}, {"name": "feature", "value": "Accept Order API"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_accept_order"}, {"name": "subSuite", "value": "TestAcceptOrder"}, {"name": "host", "value": "HOME-PC"}, {"name": "thread", "value": "9396-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_accept_order"}]}